fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -43,7 +43,15 @@ def self.bundler?
     # Usually this means the fastlane directory is ~/.fastlane/bin/
     # We set this value via the environment variable `FASTLANE_SELF_CONTAINED`
     def self.contained_fastlane?
-      ENV["FASTLANE_SELF_CONTAINED"].to_s == "true"
+      ENV["FASTLANE_SELF_CONTAINED"].to_s == "true" && !self.is_homebrew?
+    end
+
+    def self.is_homebrew?
```

Comment:
Should be called `self.homebrew?` as rubocop will complain otherwise. We just have the `is_something?` for legacy reasons around, calling the actual method 👍